content/source/docs/cloud/paid.html.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: "cloud"
+page_title: "Free and Paid Plans - Terraform Cloud"
+---
+
+[owners]: ./users-teams-organizations/teams.html#the-owners-team
+
+# Free and Paid Plans
+
+Terraform Cloud is a commercial SaaS product developed by HashiCorp. We make many of its features available for free, and also sell paid upgrade packages.
```

Comment:
> and also sell paid upgrade packages.
Would suggest "and also offer additional collaboration and governance features as part of a few paid plans"